osquery/tables/system/linux/shadow.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <shadow.h>
+
+#include <boost/regex.hpp>
+
+#include <mutex>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+
+namespace osquery {
+namespace tables {
+
+Mutex spwdEnumerationMutex;
+
+void genShadowForAccount(const struct spwd* spwd, QueryData& results) {
+  Row r;
+  r["last_change"] = BIGINT(spwd->sp_lstchg);
+  r["min"] = BIGINT(spwd->sp_min);
+  r["max"] = BIGINT(spwd->sp_max);
+  r["warning"] = BIGINT(spwd->sp_warn);
+  r["inactive"] = BIGINT(spwd->sp_inact);
+  r["expire"] = BIGINT(spwd->sp_expire);
+
+  if (spwd->sp_namp != nullptr) {
+    r["username"] = TEXT(spwd->sp_namp);
+  }
+
+  if (spwd->sp_pwdp != nullptr) {
+    const auto kLockedPasswordRegex = boost::regex("^[\!\*x]([\*\!])?");
```

Comment:
We tend to prefer the `const` with `k` prefix to denote a global constant value. Would you mind either bringing these variables up to the `tables` scope or drop the prefix?